### DISCUSSION TITLE
endless loop trying to confirm Budibase account

### DISCUSSION THREAD

**Discussion by LockeClone** [Budibase/budibase#4260] [Help]
Well, we've got a failure to launch here...

I've signed up, but whenever I click the link in my verification email I get a quick message sating "Error creating tenant" while being dumped back onto the "check your email page" where I have the option to resend email. Where we start the process over again.

Any insight? I'd really like to use the software.

**Comment by pavleks** [comment]
Hi @LockeClone try to follow this [discussion](https://github.com/Budibase/budibase/discussions/3734#discussioncomment-1807255), it might help.

**Reply by LockeClone** [reply]
Thank you. I didn't try to use a number but I am an incessant capitalizer. I wonder if having a capital letter at the beginning could have gotten me here?

**Reply by pavleks** [reply]
I believe yes. I just created a new user under a different email and created a domain name `Alex.budibase.com` (capital A). Getting the same error message.

**Reply by mjashanks** [reply]
Awesome  - getting this fixed asap. Thank you!

**Comment by mjashanks** [comment]
@LockeClone would you be able to post your domain name, or send it to support@budibase.com ?

i.e. `https://your-domain.budibase.app`

Thanks!

**Reply by LockeClone** [reply]
I believe it's "Gravityworks.budibase.app" with a capital "G". I unfortunately didn't write it down. I'll email the support address you gave me as well. Thank you.

**Reply by mjashanks** [reply]
@LockeClone we've fixed this now. You should be able to login now via https://account.budibase.app or https://gravityworks.budibase.app

We no longer allow uppercase in tenant names.

**Reply by LockeClone** [reply]
Well that was very responsive. Thank you much!
